src/shared/ui/survey/MediaItems/MediaInput.tsx

```diff
@@ -61,7 +69,7 @@ const MediaInput: FC<Props> = ({
   };
 
   return (
-    <ContentWrapper height={windowWidth * 0.85}>
+    <ContentWrapper hasValue={hasValue} height={windowWidth * 0.85}>
```

Comment:
Can we just use `borderColor` property right here? I don't think there is a need to create a variant for the sake of one style property.

Reply:
Please make `MediaInput` accept BoxProps and just pass these styled props to `ContentWrapper`.
You will decide which color is passed to `MediaInput` in the parent component (PhotoItem, VideoItem)